api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -507,6 +516,18 @@ message Features {
   // Policy enables the Teleport Policy feature set.
   // At the time of writing, this includes Teleport Access Graph (TAG).
   PolicyFeature Policy = 28 [(gogoproto.jsontag) = "policy,omitempty"];
+  // Questionnaire indicates whether cluster users should get an onboarding questionnaire
+  bool Questionnaire = 29 [(gogoproto.jsontag) = "questionnaire,omitempty"];
+  // IsStripeManaged indicates if the cluster billing is managed via Stripe
+  bool IsStripeManaged = 30 [(gogoproto.jsontag) = "isStripeManaged,omitempty"];
```

Comment:
Should we be going snake_case like the others fields?

is_stripe_managed, external_audit_storage, support_type, join_active_sessions, mobile_device_management